e2e/_suites/ingest-manager/features/agent_endpoint_integration.feature

```diff
@@ -0,0 +1,38 @@
+@agent_endpoint_integration
+Feature: Fleet Mode Agent with Endpoint Integration
```

Comment:
```suggestion
Feature: Agent Endpoint Integration
```
The best practices say that the file must be named after the `Feature`, or viceversa